src/index.js

```diff
@@ -0,0 +1 @@
+import './style.css';
```

Comment:
## NICE APP 👏
- [ ] You did great job with installing webpack successfully. Everything is working as expected ⚡👍. Please add some javascript functionality and make it work a little. You must use ES6 javascript like `arrow functions`, `let `and `const `varaibles. App should not work 100% yet because you will add things with APIs in the following lessons 🍀😉

Reply:
Ok..........Looking into it